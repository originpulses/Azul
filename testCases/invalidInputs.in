this is an input
quit
7
9
3
6
d
4