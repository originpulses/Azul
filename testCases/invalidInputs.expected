Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit

> Invalid Input, Enter again
> Invalid Input, Enter again
> Invalid Input, Enter again
> Invalid Input, Enter again
> 	[48;2;0;255;34;30m ================================[m

	[1;37;41mDeveloped By: Syed Sabih Ali[m

	[1;37;41mStudent ID: s3528350[m

	[48;2;0;255;34;30m ================================[m

Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit

> Invalid Input, Enter again
> 

Goodbye
